tools/clang/lib/SPIRV/TypeTranslator.cpp

```diff
@@ -965,8 +965,9 @@ TypeTranslator::getAlignmentAndSize(QualType type, LayoutRule rule,
 
     for (const auto *field : structType->getDecl()->fields()) {
       uint32_t memberAlignment = 0, memberSize = 0;
+    const bool isRowMajor = field->hasAttr<HLSLRowMajorAttr>() || !field->hasAttr<HLSLColumnMajorAttr>() && spirvOptions.HLSLDefaultRowMajor;
```

Comment:
I think we also need to fix the appearance in `translateType()`:
https://github.com/Microsoft/DirectXShaderCompiler/blob/578b0df014c56cbf1aba631ad35e6ea13face358/tools/clang/lib/SPIRV/TypeTranslator.cpp#L254-L255

Reply:
This is not fixed yet?